Loop/Views/Presets/PresetRangeEditor.swift

```diff
@@ -0,0 +1,186 @@
+                        .aspectRatio(contentMode: .fit)
+                        .foregroundColor(.accentColor)
+                        .frame(width: UIFontMetrics.default.scaledValue(for: 14), height: UIFontMetrics.default.scaledValue(for: 14))
+                }
+                .padding(.top, 10)
+
+
+                Text("Set your correction range")
+                    .font(.title2)
+                    .fontWeight(.bold)
+                    .multilineTextAlignment(.center)
+                    .padding(.top, 10)
+
+                Text("To reduce the risk of highs or lows, you may want to set an adjusted range if you think your glucose will vary more than usual.")
+                    .multilineTextAlignment(.center)
+
+                if allowsScheduledRange {
+                    Toggle("Use Scheduled Range", isOn: Binding(get: {
+                        range == nil
+                    }, set: { newValue in
+                        withAnimation {
+                            if (newValue) {
+                                range = nil
+                            } else {
+                                range = scheduledRange
+                            }
+                        }
+                    }))
+                    .padding(.vertical)
+                    .font(.system(size: 17))
+                }
+            }
+
+            VStack(spacing: 0) {
+                if (range == nil) {
+                    Text("Currently Scheduled Correction Range")
+                } else {
+                    Text("Adjusted Range")
+
+                }
+
+                (
+                    boundText(for: (displayedRange).lowerBound) +
+                    Text("-").foregroundColor(.secondary)
+                        .font(.system(size: 42, weight: .light))
+                    +
+                    boundText(for: (displayedRange).upperBound)
+                )
+
+
+                Text("mg/dL")
+                    .foregroundColor(.secondary)
+            }
+
+            if range != nil {
+                Divider()
+                    .animation(.default, value: range != nil)
+
+                GlucoseRangePicker(range: Binding(
+                    get: { displayedRange },
+                    set: { range = $0 }),
+                                   unit: displayGlucosePreference.unit,
+                                   minValue: nil,
+                                   guardrail: guardrail)
+                .padding(.vertical, -20)
+            }
+
+            HStack(spacing: 8) {
+                Image(systemName: "info.circle")
+                    .foregroundColor(.accentColor)
+
+                tipText.font(.system(size: 14))
+            }
+            .padding()
+            .overlay( /// apply a rounded border
+                RoundedRectangle(cornerRadius: 8)
+                    .stroke(.gray, lineWidth: 1)
+            )
+            .padding(.bottom)
+        }
+        .font(.system(size: 15))
```

Comment:
`.subheadline`?

Here's this if it's helpful: https://gist.github.com/zacwest/916d31da5d03405809c4